src/app/components/ModalContainer/pokemonSearchDetail/index.tsx

```
@@ -62,7 +72,25 @@ const PokemonSearchDetail = ({ pokemonData }: PokemonDetailProps) => {
+          </Typography>
+          <div>
+            {abilityJaFilter &&
+              abilityJaFilter.map((ability) => (
+                <li key={ability.name}>
+                  <Typography color="black" variant="p">
+                    {ability.name}
+                  </Typography>
+                </li>
+              ))}
+          </div>
+        </div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)